addons/website_sale/security/ir.model.access.csv

```diff
@@ -23,3 +23,5 @@ access_product_image_sale,product.image sale,model_product_image,sales_team.grou
 access_website_sale_payment_acquirer_onboarding_wizard,access.website.sale.payment.acquirer.onboarding.wizard,model_website_sale_payment_acquirer_onboarding_wizard,base.group_system,1,1,1,0
 access_ecom_extra_fields_public,access_ecom_extra_field public,model_website_sale_extra_field,,1,0,0,0
 access_ecom_extra_fields_publisher,access_ecom_extra_field publisher,model_website_sale_extra_field,website.group_website_publisher,1,1,1,1
+access_website_base_unit_public,website.base.unit public,model_website_base_unit,,1,0,0,0
+access_website_base_unit_sale,website.base.unit sale,model_website_base_unit,sales_team.group_sale_manager,1,1,1,1
```

Comment:
`access_website_base_unit_sale_mananger` or `access_website_base_unit_manager` instead of `access_website_base_unit_sale` ?